src/pages/home/Home.tsx

```diff
@@ -90,9 +90,9 @@ export default function home() {
             <StyledCtaBar className='container'>
               <div className='row' >
                 <div className='col-md-6 mt-2'>
-                  <Text variant='light' size={24}>Ready to get started?</Text>
+                  <PageSubTitle variant='light'>Ready to get started?</PageSubTitle>
                 </div>
-                <div className='col-md-6 text-right'>
+                <div className='col-md-6 text-right'  style={{paddingTop: '12px'}} >
                   <Button label='Get Started' />
```

Comment:
above you are using <button> but here you are using <Button>. Please keep it consistent, only use <Button> avoid using <button> from now on.